plugins/c8y_configuration_plugin/src/main.rs

```diff
@@ -71,10 +71,7 @@ async fn create_mqtt_client(mqtt_port: u16) -> Result<mqtt_channel::Connection,
         mqtt_channel::TopicFilter::new_unchecked(C8yTopic::SmartRestRequest.as_str());
     topic_filter.add_all(health_check_topics("c8y-configuration-plugin"));
 
-    topic_filter.add_all(ConfigResponseTopic::SnapshotRequest.into());
-    topic_filter.add_all(ConfigResponseTopic::SnapshotResponse.into());
-    topic_filter.add_all(ConfigResponseTopic::UpdateRequest.into());
-    topic_filter.add_all(ConfigResponseTopic::UpdateResponse.into());
+    topic_filter.add_all(ConfigOperationTopic::SnapshotRequest.into());
```

Comment:
You need to subscribe to both responses topics and not the request topics.
```suggestion
    topic_filter.add_all(ConfigOperationTopic::SnapshotResponse.into());
    topic_filter.add_all(ConfigOperationTopic::UpdateResponse.into());
```